examples/twittermap/web/app/websocket/WebSocketFactory.scala

```diff
@@ -5,10 +5,10 @@ import org.eclipse.jetty.websocket.client.WebSocketClient
 
 class WebSocketFactory {
 
-  def newClient(): WebSocketClient = {
+  def newClient(maxTextMessageSize: Int): WebSocketClient = {
     val socket = new WebSocketClient()
-    //TODO Should be configurable in later PR.
-    socket.setMaxTextMessageBufferSize(5*1024*1024)
+    socket.setMaxTextMessageBufferSize(maxTextMessageSize)
+    socket.getPolicy.setMaxTextMessageSize(maxTextMessageSize)
```

Comment:
What's the difference between these two lines? Just curious.

Reply:
http://www.eclipse.org/jetty/javadoc/9.4.4.v20170414/org/eclipse/jetty/websocket/api/WebSocketPolicy.html#setMaxTextMessageBufferSize-int-

Seems the `TextMessageSize` is what we want to configure. But I don't have large data set to test so I keep the two changes we had previously.